core/src/dird/ua_output.cc

```diff
@@ -1384,15 +1384,17 @@ RunResource* find_next_run(RunResource* run,
       wom = mday / 7;
       woy = TmWoy(future);
 
-      is_scheduled = BitIsSet(mday, run->mday) && BitIsSet(wday, run->wday) &&
-                     BitIsSet(month, run->month) && BitIsSet(wom, run->wom) &&
-                     BitIsSet(woy, run->woy);
+      is_scheduled = BitIsSet(mday, run->date_time_bitfield.mday) &&
```

Comment:
```suggestion
      bool is_scheduled = BitIsSet(mday, run->date_time_bitfield.mday) &&
```
and remove the int declaration above